### ISSUE TITLE
[Trust] After 12MAY2022 client update trusts not populating properly in party menu [Commit e9ee327]

### ISSUE THREAD

**Issue by kaincenteno** [DerpyProjectGroup/topaz#1713]
- [x] I have paid attention to this example and will edit again if need be to not break the formatting, or I will be ignored
- [x] I have searched existing [issues](https://github.com/LandSandBoat/server/issues) to see if the issue has already been opened
- [x] I have checked the commit log to see if the issue has been resolved since my server was last updated
- [x] I have read the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md)
- [x] I have specified what branch this happens on branch: base

**_Additional Information_** (Steps to reproduce/Expected behavior) **:** 

![image](https://user-images.githubusercontent.com/26943220/168147617-9768495c-b897-47e0-8668-a416016efe15.png)

**Comment by CatsEyeXI** [comment]
Even without trusts in the group (solo), my name will appear in the party list multiple times down beneath my view, much like yours does.

**Comment by GodModeXI** [comment]
Same issues here, but in addition to seeing trust and name duplication, I walk around with no hp or mp.
![image](https://user-images.githubusercontent.com/99501957/168204761-a3f37321-272b-43af-a8cf-fa382953aa10.png)

**Comment by RAIST5150** [comment]
After a death warp and returning to the zone, party list started behaving again?

Curious if it may be related to unloading tparty.  I have not yet disabled it from loading by default, but plan on doing so when I get home tonight.  Things seem to have stabilized somewhat by unloading tparty and only using 4 trusts (primarily tank and melee).

**Comment by GodModeXI** [comment]
> After a death warp and returning to the zone, party list started behaving again?
> 
> Curious if it may be related to unloading tparty. I have not yet disabled it from loading by default, but plan on doing so when I get home tonight. Things seem to have stabilized somewhat by unloading tparty and only using 4 trusts (primarily tank and melee).

- what happens when you add a 5th trust?
-  I didn’t death warp but I reloaded and zoned, I also took the time to wipe and pull the repository and set up from scratch again. And still had similar issues. I logged onto omicron and and saw similar conditions from the user perspective. Nocturnal souls and caldera have no issues. Retail is working correctly. All through ashita. I’ll unload tParty tonight and take a gander, thank you for the input.

**Comment by RAIST5150** [comment]
5th trusts have also been shown to trigger issues.  It may just crash the client if you have a pet out... but have also seen the server go down when you start pulling mobs when on a non pet job.

The reason why I have started unloading tparty is to stop the spam in the logs that it generates, considering we seem to be having an issue of sorts with SQL query overhead/overload.

Also with us sometimes triggering crashes in10-30 minutes trying to narrow things down, there has been a lot of crud for Kain to sift through... anything to reduce bloat in the logs.

**Comment by drobinsona149** [comment]
Duly noted. Thank you

**Comment by GodModeXI** [comment]
![image](https://user-images.githubusercontent.com/99501957/168448821-9cc202e4-66ae-42a0-b7a0-be4a9d1a598d.png)
So I went back and did a file check right after updating retail to current, but there was consistently 108 corrupt files even after repair. I completely wiped retail and reinstalled the whole thing from scratch. When I loaded back in, I still had the issue, BUT when I had a trust out and hit the party disband button, it fixed my issue.
